### ISSUE TITLE
Checkbox and radio focus outlines

### ISSUE THREAD

**Issue by teenwolfblitzer** [Kajabi/sage-archived#221]
## Description
We'll need to investigate overriding the existing `:focus` styling inherited from Bootstrap forms in `kajabi-products`.

- [ ] checkbox inputs
- [ ] radio inputs
- [ ] switches

Because of default `:focus` styles being applied to base elements `[type="checkbox"]` and `[type="radio"]`, the Sage focus outlines are not visible (see screenshot below):

![checkbox-focus](https://user-images.githubusercontent.com/816579/80849638-e16fa100-8bcc-11ea-8482-a5c05320444a.png)

A probable fix would be to reset `outline: none` on all `:focus` and `:active` styles for each of the elements.
